configs/s2ef/all-forceonly/forcenet/forcenet_basic.yml

```diff
@@ -0,0 +1,43 @@
+trainer: forces
+
+dataset:
+  - src: /checkpoint/electrocatalysis/relaxations/features/struct_to_energy_forces/train/200k/
+    normalize_labels: True
+    target_mean: -0.7554450631141663
+    target_std: 2.887317180633545
+    grad_target_mean: 0.0
+    grad_target_std: 2.887317180633545
+  - src: /checkpoint/electrocatalysis/relaxations/features/struct_to_energy_forces/val/is_30000/
+
+model:
+  name: forcenet
```

Comment:
Also a good idea to define the model parameters here for people to know what flags exist without looking through the source code.

Reply:
Yes, I will add the default params explicitly.

Reply:
added default params in forceonly yml